lib/screen/detail/artwork_detail_page.dart

```diff
@@ -208,160 +250,270 @@ class _ArtworkDetailPageState extends State<ArtworkDetailPage>
+                          child: IconButton(
+                            onPressed: () => Navigator.pop(context),
+                            constraints: const BoxConstraints(
+                              maxWidth: 34,
+                              maxHeight: 34,
+                            ),
+                            icon: SvgPicture.asset(
+                              'assets/images/ff_back_dark.svg',
+                            ),
+                            padding: const EdgeInsets.all(0),
+                          ),
+                        ),
+                        centerTitle: false,
+                        backgroundColor: Colors.transparent,
+                        actions: [
+                          FFCastButton(
+                            onCastTap: () async => _onCastTap(asset),
+                            isCasting: isCasting,
+                          ),
+                        ],
                       ),
                     ),
                   ),
-                ),
-                const SizedBox(
-                  height: 40,
-                ),
-                Visibility(
-                  visible: editionSubTitle.isNotEmpty,
-                  child: Padding(
-                    padding: ResponsiveLayout.getPadding,
-                    child: Text(
-                      editionSubTitle,
-                      style: theme.textTheme.ppMori400Grey14,
-                    ),
-                  ),
-                ),
-                debugInfoWidget(context, currentAsset),
-                const SizedBox(height: 16),
-                Padding(
-                  padding: ResponsiveLayout.getPadding,
-                  child: Column(
-                    crossAxisAlignment: CrossAxisAlignment.start,
+                  body: Stack(
                     children: [
-                      Semantics(
-                        label: 'Desc',
-                        child: HtmlWidget(
-                          customStylesBuilder: auHtmlStyle,
-                          asset.description ?? '',
-                          textStyle: theme.textTheme.ppMori400White14,
-                        ),
+                      Column(
+                        crossAxisAlignment: CrossAxisAlignment.start,
+                        children: [
+                          Expanded(
+                            child: Hero(
+                              tag: 'detail_${asset.id}',
+                              child: _ArtworkView(
+                                payload: widget.payload,
+                                token: asset,
+                                focusNode: _focusNode,
+                              ),
+                            ),
+                          ),
+                          const Padding(
+                            padding: EdgeInsets.symmetric(vertical: 15),
+                            child: ArtworkDetailsHeader(
+                              title: 'I',
```

Comment:
why "I"?

Reply:
Just random text to make sure it has the same heigh